src/data/icons/svg/jCHF.svg

```diff
@@ -0,0 +1,9 @@
+<svg id="Group_1831" data-name="Group 1831" xmlns="http://www.w3.org/2000/svg" width="145" height="145" viewBox="0 0 145 145">
+  <circle id="Ellipse_73" data-name="Ellipse 73" cx="72.368" cy="72.368" r="72.368" transform="translate(0 0)" fill="#fff"/>
+  <path id="Ellipse_73_-_Outline" data-name="Ellipse 73 - Outline" d="M72.5,6.042a66.458,66.458,0,1,0,46.993,19.465A66.458,66.458,0,0,0,72.5,6.042M72.5,0A72.5,72.5,0,1,1,0,72.5,72.5,72.5,0,0,1,72.5,0Z" transform="translate(0 0)" fill="#d80027"/>
+  <circle id="Ellipse_78" data-name="Ellipse 78" cx="60.214" cy="60.214" r="60.214" transform="translate(12.153 12.153)" fill="#d80027"/>
+  <g id="Group_21" data-name="Group 21" transform="translate(29.284 29.282)">
```

Comment:
Thanks for the contribution. could you merge master in? we have extra test that do some checks to ensure icons are fine. Thanks

Unfortunately, we can't afford to have `id=`, `class=` or `<style>` in these SVG because we will convert them back into react class.

Reply:
Hi @gre , do you want me to merge the branch in master and reopen the PR from our master as I didn't fully understand?

I'll see with our designer to address the issues mentioned with `id=`, `class=` and `<style>` and will make a commit with the new files :)